docs/tutorials/kafka-connect-replicator.rst

```diff
@@ -0,0 +1,402 @@
+.. _kafka_connect_replicator :
+
+Kafka Connect Replicator
+--------------------
+
+In this section, we provide a tutorial for running Kafka Connect Replicator which replicates data from two source Kafka clusters to a destination Kafka cluster.  By the end of this tutorial, you will have successfully run Kafka Connect Replicator and replicated  data for two topics from different source clusters to a destination cluster.
+
+
+It is worth noting that we will be configuring Kafka and Zookeeper to store data locally in the Docker containers.  For production deployments (or generally whenever you care about not losing data), you should use mounted volumes for persisting data in the event that a container stops running or is restarted.  This is important when running a system like Kafka on Docker, as it relies heavily on the filesystem for storing and caching messages.  Refer to our `documentation on Docker external volumes <operations/external-volumes.html>`_ for an example of how to add mounted volumes to the host machine.
```

Comment:
Probably should be a little more formal here "For production deployments (or generally whenever you care about not losing data)"
maybe change to "For deployments that require persistent data (e.g. production deployments)"